airflow/www/extensions/init_views.py

```diff
@@ -220,6 +220,27 @@ def _handle_method_not_allowed(ex):
     app.extensions["csrf"].exempt(api_bp)
 
 
+def init_api_internal(app: Flask) -> None:
+    """Initialize Internal API"""
+    if not conf.getboolean("webserver", "run_internal_api", fallback=False):
+        return
+    base_path = "/internal/v1"
```

Comment:
I have a feeling this should be a bit more descriptive URL: `/internal_api/v1` ? - same as open api specification basically (`internal_api_v1.yaml`),